container_linux.go

```diff
@@ -213,6 +215,12 @@ func newPipe() (parent *os.File, child *os.File, err error) {
 func (c *linuxContainer) Destroy() error {
 	c.m.Lock()
 	defer c.m.Unlock()
+	// Since the state.json and CRIU image files are in the c.root
+	// directory, we should not remove it after checkpoint.  Also,
+	// when     CRIU exits after restore, we should not kill the processes.
+	if _, err := os.Stat(filepath.Join(c.root, "checkpoint")); err == nil {
```

Comment:
I have question about `c.root`, because now we supposing that all in that root is not critical and can be removed on reboot or containers cleaning. But c/r data is pretty critical, so it should be persisted, probably makes sense to add `CRIURoot`?